doc/user-guide/cli.md

```diff
@@ -428,12 +446,14 @@ ploomber-cloud labels --sync
 
 By default, the CLI reads and writes to a `ploomber-cloud.json` file, but you can customize it via the `--config` (or its short version, `-c`), switching the config file is useful when you need to manage multiple environments (for example, development and production).
 
-All commands that read or write the config file accept the `--config/-c` argument, for example:
+All commands that read or write the config file accept the `--config/-c` argument. You can use this option to specify both alternative file names and paths, for example:
 
 ```sh
 # create the config file in ploomber-cloud.dev.json
 ploomber-cloud init --config ploomber-cloud.dev.json
 
 # deploy using the dev config
-ploomber-cloud deploy --config ploomber-cloud.dev.json
+ploomber-cloud deploy --config ./config/ploomber-cloud.dev.json
```

Comment:
Why did we add `./config/..`?

Reply:
```suggestion
ploomber-cloud deploy --config ploomber-cloud.dev.json
```

Reply:
fix: https://github.com/ploomber/doc/pull/287/commits/39b8ba26e5be7e7a96ae1353dae0b0b36bd29b14